src/main.js

```diff
@@ -25,6 +25,7 @@ let docs                = require('taskcluster-lib-docs');
 let App                 = require('taskcluster-lib-app');
 let remoteS3            = require('remotely-signed-s3');
 let {sasCredentials}    = require('taskcluster-lib-azure');
+let {Client, pulseCredentials} = require('taskcluster-lib-pulse');
```

Comment:
The name `Client` is not the most specific name here.  Can this constructor be renamed to something like`PulseClient`?

Reply:
It can be referenced as `pulse.Client` if `const pulse = require('taskcluster-lib-pulse')`.